doc/installing.rst

```diff
@@ -38,36 +38,153 @@ System Requirements
 .. note::
    For Linux distributions, we support, if technically feasible, the latest 2 versions per platform and the previous Ubuntu `LTS`_.
 
+Customizing the Windows installation
+----------------------------------
+
+If you just want to install ownCloud Desktop Synchronization Client on your local
+system, you can simply launch the .msi file and configure it in the wizard
+that pops up.
+
+Features
+^^^^^^^^
+
+The MSI installer provides several features that can be installed or removed
+individually, which you can also control via command-line, if you are automating
+the installation::
+
+   msiexec /passive /i ownCloud-x.y.z.msi
+
+will install the ownCloud Desktop Synchronization Client into the default location
+with the default features enabled. If you want to disable e.g. desktop shortcut
+icons you can simply change the above command to::
+
+   msiexec /passive /i ownCloud-x.y.z.msi REMOVE=DesktopShortcut
+
+See the following table for a list of available features:
+
++--------------------+--------------------+----------------------------------+---------------------------+
+| Feature            | Enabled by default | Description                      |Property to disable        |
++====================+====================+==================================+===========================+
+| Client             | Yes, required      | The actual client                |                           |
++--------------------+--------------------+----------------------------------+---------------------------+
+| DesktopShortcut    | Yes                | Adds a shortcut to the desktop   |`NO_DESKTOP_SHORTCUT`      |
++--------------------+--------------------+----------------------------------+---------------------------+
+| StartMenuShortcuts | Yes                | Adds shortcuts to the start menu |`NO_START_MENU_SHORTCUTS`  |
++--------------------+--------------------+----------------------------------+---------------------------+
+| ShellExtensions    | Yes                | Adds Explorer integration        |`NO_SHELL_EXTENSIONS`      |
++--------------------+--------------------+----------------------------------+---------------------------+
+
+**Installation**
```

Comment:
How about setting this, and similar text, as a level-3 header by underlining it with a tilde?